tasks/testTasks.ts

```diff
@@ -1,14 +1,21 @@
 import gulp from 'gulp';
+import os from 'os';
 import path from 'path';
 import simpleGit, { CleanOptions } from 'simple-git';
 import { runTests } from 'vscode-test';
 
-import { defaultOutputPath,rootPath, testAssetsPath } from './projectPaths';
+import { defaultOutputPath, rootPath, testAssetsPath } from './projectPaths';
 
 // eslint-disable-next-line node/no-missing-require
 require('./benchmarkTestTask');
 
 gulp.task('test:e2e', async () => {
+    // TODO: remove this after vscode fixes the issue that the deactive function is not called on mac

```

Comment:
Let's add the issue link(https://github.com/microsoft/vscode/issues/114688) here for easier track